app/screens/send-bitcoin-screen/send-bitcoin-success-screen.tsx

```diff
@@ -11,37 +11,95 @@ import { RootStackParamList } from "@app/navigation/stack-param-lists"
 import { useNavigation } from "@react-navigation/native"
 import { StackNavigationProp } from "@react-navigation/stack"
 import { makeStyles, Text } from "@rneui/themed"
-import { View } from "react-native"
+import { View, Alert } from "react-native"
 import { testProps } from "../../utils/testProps"
-
-const useStyles = makeStyles(() => ({
-  contentContainer: {
-    flexGrow: 1,
-  },
-  successText: {
-    fontSize: 18,
-    textAlign: "center",
-    marginTop: 20,
-  },
-  Container: {
-    flex: 1,
-    justifyContent: "center",
-    alignItems: "center",
-  },
-}))
+import Rate from "react-native-rate"
+import { ratingOptions } from "@app/config"
+import crashlytics from "@react-native-firebase/crashlytics"
+import { useApolloClient } from "@apollo/client"
+import { useFeedbackModalShownQuery } from "@app/graphql/generated"
+import { setFeedbackModalShown } from "@app/graphql/client-only-query"
+import { SuggestionModal } from "./suggestion-modal"
+import { logAppFeedback } from "@app/utils/analytics"
 
 const SendBitcoinSuccessScreen = () => {
   const styles = useStyles()
+  const [isFeedbackModalActive, setIsFeedbackModalActive] = React.useState(false)
```

Comment:
It seems like `isFeedbackModalActive` is unnecessary. In line 65 you can just call a function that calls `Alert.alert` directly instead of using a `useEffect`.